lib/token_authenticate_me/concerns/controllers/authenticateable.rb

```diff
@@ -29,7 +29,7 @@ def authenticate
 
         def current_user
           return unless authenticated_session
-          @current_user ||= User.find_by_id(authenticated_session.user_id)
+          @current_user = authenticated_session.user
```

Comment:
No need to set a reference to it, lets just return the user
```suggestion
          authenticated_session.user
```